tool/site/client-react/tests/pages/__snapshots__/loginPage.spec.tsx.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Displays user/system options in internal-dev 1`] = `
```

Comment:
delete the change-auth snap?

Reply:
there was no change-auth snap as there were no tests covering this area before I added them

Reply:
lol :clown_face: